### PR TITLE
🚧Fix nuget package check issue

### PR DESCRIPTION
**Pull request by CalvinWilkinson**
<h1 style="font-weight:bold" align="center">Preview Feature Pull Request</h1>
<h2 style="font-weight:bold" align="center">📋Review Checklist📋</h2>

- [x] The **_head(source)_** branch for this pull request is a **_preview feature_** branch, with the correct naming syntax, in the following example:
  - A **_head(source)_** branch is the branch being merged into a target branch.
    <details closed><summary>Example</summary>

      ``` xml
      Syntax: preview/feature/<issue-num>-<description>
      Example: preview/feature/123-my-preview-feature
      ```
    </details>

- [x] The **_head(source)_** branch for this pull request is created from a **_preview release_** branch, with the correct naming syntax, in the following example:
  <details closed><summary>Example</summary>

    ``` xml
    Syntax: preview/v<major>.<minor>.<patch>-preview.<prev-num>
    Example: preview/v1.2.3-preview.4
    ```
  </details>

- [x] The **_base(target)_** branch for this pull request is a **_preview release_** branch, with the correct naming syntax, in the following example:
  - A **_base(target)_** branch is the branch that the **_head(source)_** branch is merging into.

    <details closed><summary>Example</summary>

      ``` xml
      Syntax: preview/v<major>.<minor>.<patch>-preview.<prev-num>
      Example: preview/v1.2.3-preview.4
      ```
    </details>

💡For more information on branching, refer to the project's [branching documentation](../../Documentation/Branching.md).

- [x] Associated issue exists and is linked to this pull request.
  - One issue per pull request.

- [x] Pull request title matches the title of the associated issue.

- [x] A `preview` label has been added to the pull request.

- [x] The labels attached to this PR match the labels attached to the associated issue.

- [x] Project assigned matches the project of the associated issue.

- [x] Milestone assigned matches the milestone of the associated issue.

- [x] My code follows the coding style of this project.
  - The style is enforced by the **_.editorconfig_** files in the project and displayed as warnings.  If there is an edge case, reach out, and let's discuss it.

- [x] I have manually tested my code changes to the best of my ability.

- [x] All tests passed locally.
  - This is required because unit tests might pass locally but not in the CI environment during the status check process or vice-versa.
  - Tests might pass on the developer's machine but not necessarily on the code reviewer's machine.
  - Status checks run for this pull request when merging into the **_preview/v\*.\*.\*-preview.\*_** branch.  These status checks run every time a change is pushed to the pull request.  These checks validate version syntax, tagging, builds, unit tests, and more.

### CODE REVIEW

File: CICD.sln.DotSettings [hunk 1]
```diff
@@ -90,13 +90,17 @@ private $OBJECT_TYPE$ CreateSystemUnderTest()
...
+	<s:Boolean x:Key="/Default/UserDictionary/Words/=prerelease/@EntryIndexedValue">True</s:Boolean>
 	<s:Boolean x:Key="/Default/UserDictionary/Words/=pwsh/@EntryIndexedValue">True</s:Boolean>
 	<s:Boolean x:Key="/Default/UserDictionary/Words/=Reactable/@EntryIndexedValue">True</s:Boolean>
 	<s:Boolean x:Key="/Default/UserDictionary/Words/=Reactables/@EntryIndexedValue">True</s:Boolean>
 	<s:Boolean x:Key="/Default/UserDictionary/Words/=Unsubscriber/@EntryIndexedValue">True</s:Boolean>
 	<s:Boolean x:Key="/Default/UserDictionary/Words/=URL_0027s/@EntryIndexedValue">True</s:Boolean>
-	<s:Boolean x:Key="/Default/UserDictionary/Words/=Usings/@EntryIndexedValue">True</s:Boolean></wpf:ResourceDictionary>
\ No newline at end of file
+	<s:Boolean x:Key="/Default/UserDictionary/Words/=Usings/@EntryIndexedValue">True</s:Boolean>
+	<s:Boolean x:Key="/Default/UserDictionary/Words/=VVVVV/@EntryIndexedValue">True</s:Boolean></wpf:ResourceDictionary>
```
Comment (KinsonDigitalAdmin) [inline CICD.sln.DotSettings:106]: Add a single empty line to the end of the file.

File: CICD/ExtensionMethods.cs [hunk 2]
```diff
@@ -252,6 +267,26 @@ public static BranchType GetBranchType(this string branch)
         return BranchType.Other;
     }
 
+    public static string ToCrossPlatPath(this string value)
```
Comment (KinsonDigitalAdmin) [inline CICD/ExtensionMethods.cs:270]: Add code docs

File: CICD/ExtensionMethods.cs [hunk 3]
```diff
@@ -252,6 +267,26 @@ public static BranchType GetBranchType(this string branch)
...
+    {
+        if (string.IsNullOrEmpty(value))
+        {
+            return string.Empty;
+        }
+
+        return value.Replace(WinDirSeparator, PosixDirSeparator);
+    }
+
+    public static string[] ToCrossPlatPaths(this IEnumerable<string>? paths)
```
Comment (KinsonDigitalAdmin) [inline CICD/ExtensionMethods.cs:280]: Add code docs

**Review by KinsonDigitalAdmin — CHANGES_REQUESTED** [review]

**Review by KinsonDigitalAdmin — DISMISSED** [review]

File: CICD/ExtensionMethods.cs [hunk 4]
```diff
@@ -252,6 +267,50 @@ public static BranchType GetBranchType(this string branch)
...
     }
 
+    /// <summary>
+    /// Converts the given path string to a cross-platform compatible path.
+    /// </summary>
+    /// <param name="value">The <c>string</c> path to convert.</param>
+    /// <returns>A cross-platform path.</returns>
+    /// <remarks>
+    /// <para>
+    ///     This simply takes all windows specific directory separator characters and
```
Comment (kselena) [inline CICD/ExtensionMethods.cs:277]: Refactor to remove simply
```suggestion
    ///     This takes all windows specific directory separator characters and
```

File: CICD/Services/Interfaces/IFindDirService.cs [hunk 5]
```diff
@@ -16,5 +16,5 @@ public interface IFindDirService
     /// <param name="startPath">The directory path to start the search.</param>
     /// <param name="dirNameToFind">The name to the directory to find.</param>
```
Comment (kselena) [inline CICD/Services/Interfaces/IFindDirService.cs:17]: Grammar
```suggestion
    /// <param name="dirNameToFind">The name of the directory to find.</param>
```

**Review by kselena — CHANGES_REQUESTED** [review]

File: CICD/Services/Interfaces/IFindDirService.cs (hunk 5 above)
Comment (kselena) [reply CICD/Services/Interfaces/IFindDirService.cs]: @CalvinWilkinson I don't see that this has been changed.

**Review by kselena — APPROVED** [review]

**Review by KinsonDigitalAdmin — APPROVED** [review]
